docs/static-typing.md

```diff
@@ -36,13 +36,13 @@ Use `Element` as the return type when you want to always return an element.
 ```python
 from typing import Literal
 
-from htpy import Element, span
+from htpy import Renderable, span
 
 
 def bootstrap_badge(
     text: str,
     style: Literal["primary", "success", "danger"] = "primary",
-) -> Element:
+) -> Renderable:
```

Comment:
this example explicitly call out the Element and VoidElement types. maybe it should be rewritten / moved down under Renderable to make it less prominent. it can be useful to know what those types are but I agree with this PR that we should put more emphasis on using Renderable

Reply:
Makes a lot of sense to keep this one as it was. I've reverted this change.